docs/UserGuide.md

```diff
@@ -76,32 +80,64 @@ Format: `help`
 
 Adds a person to the address book.
 
-Format: `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]…​`
-
+Format: `add n/NAME p/PHONE_NUMBER e/EMAIL [t/TAG]…​`
+* Names cannot be more than 100 characters long.
+* Phone numbers must be exactly 8 digits.
+* Tags must be created before they can be assigned to a person. Refer to [newtag](#creating-a-new-tag-newtag) on how to create a tag.
 <div markdown="span" class="alert alert-primary">:bulb: **Tip:**
 A person can have any number of tags (including 0)
 </div>
 
 Examples:
-* `add n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01`
-* `add n/Betsy Crowe t/friend e/betsycrowe@example.com a/Newgate Prison p/1234567 t/criminal`
+* `add n/John Doe p/98765432 e/johnd@example.com t/Groom's Friend`
+* `add n/Betsy Crowe t/Bride's Family e/betsycrowe@example.com p/12345678`
```

Comment:
Should we change Bride's Family to bride's side for more consistent tags in our examples?